api/src/test/java/uk/gov/dhsc/htbhf/claimant/testsupport/FailureEventTestDataFactory.java

```diff
@@ -14,4 +16,13 @@ public static FailureEvent aFailureEventWithEvent(Event event) {
                 .exception(TEST_EXCEPTION)
                 .build();
     }
+
+    public static FailureEvent aFailedMakePaymentEvent(PaymentCycle paymentCycle, int amountToPay, String requestReference) {
```

Comment:
Is there going to be audits for RequestPayment and CompletePayment since MakePayment has now been removed?